addons/account/static/src/components/account_move_form/account_move_form.scss

```diff
@@ -0,0 +1,8 @@
+.o_field_x2many_list_row_add {
+    a:focus-visible{
+        border-radius: 0.25rem;
+        padding: 3px 1px;
+        box-shadow: 0 0 0 #000, 0 0 0 0.25rem rgba(39, 145, 150, 0.5) !important;
+
+    }
+}
```

Comment:
@kebeclibre I showed the result in the UI to an accounting PO who liked it but we don't want to merge that kind of improvement only in Accounting to stay consistent, could you consider it for a generic improvement ?

Reply:
@aab-odoo I already validated changes in account,* with the PO, the only left open point is this one.
It adds a more visual selection when you tab in fields of lines in a list. 
Can you consider this as a general improvement for all Odoo ?
We don't want to have a different behavior for this in accounting apps only.

Reply:
@clbr-odoo We have a task aiming at improving the keynav in x2manys, and one of the issue is indeed that we don't see when the focus in on those "add" links, so you can remove this diff in your PR, we'll handle it. For the record, problems I see here:
- You can't define such global rules in account, because they apply on every x2many list accross Odoo (meaning that without account installed, they won't benefit from this style, but once account is installed, they will). So such rules should either be scoped to the invoice form view, or be moved to web.
- As you increase the padding, the width of an "add" link is larger, so it slightly moves items on its right which isn't great.

Reply:
Ok cool, then indeed let's remove it from this PR @vivp-odoo.
@aab-odoo Do you have a link to the task for traceability please ?

Reply:
https://www.odoo.com/odoo/project/133/tasks/4044404

Reply:
> Ok cool, then indeed let's remove it from this PR @vivp-odoo. @aab-odoo Do you have a link to the task for traceability please ?

Hello @clbr-odoo 
I have made the suggested changes, please have a look at it

cc @aab-odoo